hoodie-common/src/main/java/com/uber/hoodie/common/table/timeline/HoodieInstant.java

```diff
@@ -30,7 +30,19 @@
  */
 public class HoodieInstant implements Serializable {
 
-  private boolean isInflight = false;
+  /**
+   * Instant State
+   */
+  public enum State {
+    // Requested State (valid state for Compaction)
+    REQUESTED,
+    // Inflight instant
+    INFLIGHT,
+    // Committed instant
+    COMMIT
```

Comment:
same here. Having COMMIT as a state clashes with the action of an instant

Reply:
Renamed to Completed